test_vm/src/util.rs

```diff
@@ -177,54 +179,105 @@ pub fn precommit_sectors(
     while sector_idx < count {
         let msg_sector_idx_base = sector_idx;
         let mut invocs = invocs_common();
-
-        let mut param_sectors = Vec::<PreCommitSectorParams>::new();
-        let mut j = 0;
-        while j < batch_size && sector_idx < count {
-            let sector_number = sector_number_base + sector_idx;
-            param_sectors.push(PreCommitSectorParams {
-                seal_proof,
-                sector_number,
-                sealed_cid: make_sealed_cid(format!("sn: {}", sector_number).as_bytes()),
-                seal_rand_epoch: v.get_epoch() - 1,
-                deal_ids: vec![],
-                expiration,
+        if !v2 {
```

Comment:
I think we are planning to deprecate v1 in an upcoming upgrade.  If that's the case we should file an issue for cleaning up tests and link to it here to help us remember.